src/NuGet.Core/NuGet.ProjectModel/ProjectRestoreMetadata.cs

```diff
@@ -12,6 +12,10 @@ namespace NuGet.ProjectModel
 {
     public class ProjectRestoreMetadata : IEquatable<ProjectRestoreMetadata>
     {
+        public ProjectRestoreMetadata()
```

Comment:
This was useful to find where the class instances were created, but can be deleted now.